lib/sass_builder.dart

```diff
@@ -56,7 +60,10 @@ class SassBuilder implements Builder {
     var tempAssetPath = tempDir.fileFor(inputId).path;
     _log.fine('compiling file: ${tempAssetPath}');
     var cssOutput = sass.compile(tempAssetPath,
-        packageResolver: new SyncPackageResolver.root(tempDir.packagesDir.uri));
+        packageResolver: new SyncPackageResolver.root(tempDir.packagesDir.uri),
+        style: this._outputStyle == _defaultOutputStyle
+            ? sass.OutputStyle.expanded
+            : sass.OutputStyle.compressed);
```

Comment:
This behavior feels a little unexpected. If you supply anything other than the default then you get compressed? Maybe we should validate what options are being passed and give some feedback if they are not what we expect.